pkg/skaffold/util/util.go

```diff
@@ -341,3 +343,11 @@ func IsSubPath(basepath string, targetpath string) bool {
 func hasHiddenPrefix(s string) bool {
 	return strings.HasPrefix(s, hiddenPrefix)
 }
+
+// ShowHumanizeTime returns time in human readable format
+func ShowHumanizeTime(start time.Time) string {
+	if time.Since(start).Seconds() < 1 {
+		return time.Since(start).String() + " ago"
```

Comment:
I don't think it makes sense to use "ago" to indicate how long something took. If i said a deployment finished 3s ago, that means it finished in the past, but really we are printing basically when we know its done i.e. the present.

Further I think in order to accomplish just showing seconds without an excessive amount of decimals we can just use sprintf, as in this example https://play.golang.org/p/lAF0cEGMb27 and not rely on gohumanize in the first place.

Reply:
i think we can drop the `ago`. however i was under the impression humanize will round off time like 
70 seconds to something readble like 1 min 10 s

Thanks for bringing this to my attention.

Reply:
Yeah that is something nice that humanize does, it may be best instead to use [RelTime](https://pkg.go.dev/github.com/dustin/go-humanize#RelTime) instead as it will not append "ago" to the time.

Reply:
@tejal29 @IsaacPD So [RelTime](https://pkg.go.dev/github.com/dustin/go-humanize#RelTime) requires labels to be defined for function call. I was thinking maybe remove the `" ago"` string from the `humanize.Time()` return value using [TrimSuffix](https://golang.org/pkg/strings/#TrimSuffix) from the standard library. What do you think?

Reply:
You can use it like RelTime(t, time.Now(), "", "") as in this example https://play.golang.org/p/_3_lAP3RauV and that will humanly print the difference from when it started to now without any extra text like "ago" at the end.

Although this will remove any precision less than the current magnitude (i.e 2.9 seconds and 2.0 seconds will both print "2 seconds" and similarly for "1hr59min and 1hr0min" as seen here https://play.golang.org/p/zEb8noqHPmF)

Reply:
What do you think about using Round function from the time package ? ([link to doc](https://golang.org/pkg/time/#Duration.Round))

Reply:
I think it could work so long as we round down to the largest unit that is smallest than the duration, i.e.
```go
t := time.Since(start)
switch{
// i doubt rounding to days will be necessary
...
case t >= t.Hour:
      t := t.Round(t.Hour)
case t >= t.Minute:
     t := t.Round(t.Minute)
...
// etc to the smallest unit we want to show
}
```

Reply:
I'm trying to understand how the round function works here https://play.golang.org/p/xcxGWUvDppv and I've noticed that for values greater than 1 second (in  `h`,`m`,`s`)  we can round using `t.Round(time.Millisecond)` for a three point precision. Can you tell me if this is the right way to go ?

Reply:
If you do not use ```humanize.Reltime``` it would be fine, although it's similar to just using ```[Duration.Truncate](https://pkg.go.dev/time?utm_source=gopls#Duration.Truncate)``` and so we'd lose the readable spacing and expansion of units that humanize provides

If you do use ```humanize.Reltime``` this would not be sufficient since it would still return 1 hour for "1hr59min" as input, and the approach i described in the comment above (which i just edited) would be more appropriate. This is clearer here https://play.golang.org/p/nppwgVaZCso